Riot/Modules/Room/RoomViewController.m

```diff
@@ -4432,6 +4423,29 @@ - (RoomInputToolbarView*)inputToolbarViewAsRoomInputToolbarView
     return roomInputToolbarView;
 }
 
+- (void)showDifferentURLsAlertFor:(NSURL *)url visibleString:(NSString *)visibleString
```

Comment:
I would maybe call `visibleString` `visibleURLString` or `displayedURLString` instead, for clarity.